packages/fx-core/src/plugins/resource/aad/utils/configs.ts

```diff
@@ -191,27 +191,26 @@ export class SetApplicationInContextConfig {
   }
 
   public restoreConfigFromContext(ctx: PluginContext): void {
-    let frontendDomain: ConfigValue = ctx.config.get(ConfigKeys.domain);
-    if (frontendDomain) {
-      this.frontendDomain = format(frontendDomain as string, Formats.Domain);
+    let frontendDomain: ConfigValue;
```

Comment:
Previously aad plugin allows user to add "domain" key in context since fe plugin did not allow user to customize frontend endpoint, thus aad plugin will read domain first like this:
`frontendDomain = ctx.config.get(ConfigKeys.domain)`
Since now fe plugin has fixed this problem, @adashen shall we remove this part?

Reply:
As local debug will no longer consume values in env.default.json, changed to the following behavior:
- if is local debug:  then read from `localSettings.json`
- if is remote: then first check the "domain" config key